src/stylesheets/blocks/_concept.scss

```diff
@@ -30,36 +41,238 @@
   margin: 0 auto;
   max-width: rem(340);
+  width: 80vw;
+
+  @media (screen(lg)) {
+    display: inline-block;
+    margin: 0;
+  }
 }
+
+.--coffee:before {
+  background-image: url('../../images/CoffeeMug_TheConcept-01.svg');
+  height: rem(90);
+  margin-left: rem(-45);
+  top: rem(-68);
+  width: rem(85);
+
+  @media (screen(lg)) {
+    height: rem(174);
+    left: 0;
+    margin-left: rem(225);
+    top: rem(382);
+    width: rem(162);
+  }
+}
+
+.--coffee:after {
+  background-image: url('../../images/Yellow-Splatter-Bkgrd.svg');
+  bottom: 0;
+  height: rem(126);
+  margin-left: rem(50);
+  width: rem(312);
+}
+
+.--pill-bottle:before {
+  background-image: url('../../images/Bottle_Yellow_Foreground_IntroSection-01.svg');
+  height: rem(137);
+  left: 0;
+  margin-left: rem(241);
+  top: rem(382);
+  width: rem(100);
+  z-index: 1;
+}
+
+.--pill-bottle:after {
+  background-image: url('../../images/Yellow-Splatter-Bkgrd.svg');
+  bottom: 0;
+  height: rem(126);
+  margin-left: rem(50);
+  width: rem(312);
+}
+
+.--thermometer:before {
+  background-image: url('../../images/Thermometer_CheckUp-01.svg');
+  height: rem(191);
+  margin-left: rem(-85);
+  top: rem(-58);
+  width: rem(120);
+}
+
+.--thermometer:after {
+  background-image: url('../../images/SplatterBkrgd_Lilac.svg');
+  height: rem(154);
+  margin-left: rem(206);
+  top: 0;
+  width: rem(360);
+}
+
+.--pill-box:before {
+  background-image: url('../../images/BoxPills_Remember-01.svg');
+  height: rem(154);
+  left: 0;
+  margin-left: rem(108);
+  top: rem(-32);
+  width: rem(460);
+}
+
+.--pill-box:after {
+  background-image: url('../../images/SplatterBkrgd_Offwhite.svg');
+  height: rem(243);
+  left: 0;
+  top: 0;
+  width: rem(675);
+}
+
+.--money:before {
+  background-image: url('../../images/Money_FindDeals-01.svg');
+  height: rem(270);
+  left: 0;
+  margin-left: rem(188);
+  top: rem(-90);
+  width: rem(190);
+}
+
+.--money:after {
+  background-image: url('../../images/SplatterBkrgd_Mint.svg');
+  height: rem(187);
+  left: 0;
+  margin-left: rem(20);
+  top: 0;
+  width: rem(535);
+}
+
+.--good-job:before {
+  background-image: url('../../images/GoodJob-Florence.svg');
+  height: rem(85);
+  margin-left: rem(-90);
+  top: rem(-85);
+  width: rem(180);
+
+  @media (screen(lg)) {
+    bottom: 0;
+    height: rem(126);
+    left: 0;
+    margin-left: rem(130);
+    top: unset;
+    width: rem(266);
+  }
+}
+
+.--good-job:after {
+  background-image: url('../../images/SplatterBkgrd_Minty.svg');
+  bottom: rem(-20);
+  height: rem(236);
+  margin-left: rem(-10);
+  width: rem(375);
+}
+
+.--phone:before {
+  background-image: url('../../images/Telephone_KeepsYouHonest-01.svg');
+  bottom: rem(-40);
+  height: rem(220);
+  left: 0;
+  margin-left: rem(210);
+  width: rem(110);
+}
+
+.--phone:after {
+  background-image: url('../../images/SplatterBackground_Coral.svg');
+  bottom: rem(-20);
+  height: rem(234);
+  margin-left: rem(-58);
+  width: rem(455);
+}
```

Comment:
this all feels ripe for a `mixin` (https://sass-lang.com/documentation/file.SASS_REFERENCE.html#mixins)

imagining you might end up with

```scss
//scss
@mixin yourMixin {
   ...
}

.--coffee {
  @include yourMixin(some,parameters)
}
```

which would compile to

```css
/* css */
.--coffee:before {
  background-image: url('../../images/CoffeeMug_TheConcept-01.svg');
  height: rem(90);
  margin-left: rem(-45);
  top: rem(-68);
  width: rem(85);

   @media (screen(lg)) {
    height: rem(174);
    left: 0;
    margin-left: rem(225);
    top: rem(382);
    width: rem(162);
  }
}

 .--coffee:after {
  background-image: url('../../images/Yellow-Splatter-Bkgrd.svg');
  bottom: 0;
  height: rem(126);
  margin-left: rem(50);
  width: rem(312);
}
```

Reply:
in practice, I might be wrong - there's lots of customization, and the mixin might get unwieldy. but wondering if you thought about it

Reply:
i had not considered it because i was taking an individual approach to each position. they don't all use the same properties but a lot do. what would you do in this situation?